tests/unit/core/TestInstallAddon.js

```diff
@@ -1128,8 +1134,34 @@ describe(`${__filename}: withInstallHelpers`, () => {
 
       return install(addon).then(() => {
         sinon.assert.calledWith(fakeTracking.sendEvent, {
-          action: TRACKING_TYPE_EXTENSION,
-          category: INSTALL_CATEGORY,
+          action: getAction(ADDON_TYPE_EXTENSION),
+          category: getAddonEventCategory(ADDON_TYPE_EXTENSION, INSTALL_ACTION),
+          label: addon.name,
+        });
+      });
+    });
+
+    it('tracks a static theme addon install', () => {
```

Comment:
I don't see a test for tracking *the start* of a static theme add-on install.

Reply:
oh? ok i'll take a look..